Add "blockquote" to alerts description to improve SEO (#28140)
github/docs: content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md

{% note %}

**Note**: The Markdown syntax used during the beta testing period is now deprecated and will be removed. You can use the syntax as described in this section.

{% endnote %}

Alerts are an extension of the blockquote syntax that you can use to emphasize critical information. On {% data variables.product.product_name %}, they are displayed with distinctive colors and icons to indicate the importance of the content.

We recommend restricting the use of alerts to one or two per article to avoid overloading the reader. Consecutive notes should be avoided.

> Three types of alert are available. You add an alert using a special blockquote line that specifies the alert type, then specifying the alert information in a standard blockquote.

Reviewer: ```suggestion
Three types of alert are available. You can add an alert with a special blockquote line that specifies the alert type, and then add the alert information in a standard blockquote immediately after.
```

Minor: this may be slightly easier to parse?

Author: Thanks.
